File: src/Nethereum.Generators.JavaScript/codeGenNodeTests/MyContractName/DTOs/DecimalsOutputDTO.cs
using System;
using System.Threading.Tasks;
using System.Numerics;
using Nethereum.Hex.HexTypes;
using Nethereum.ABI.FunctionEncoding.Attributes;
namespace StandardToken.MyContractName.DTOs
{
    [FunctionOutput]
    public class DecimalsOutputDTO
    {
        [Parameter("uint8", "", 1)]
        public byte B {get; set;}
    }
}

File: src/Nethereum.Generators.JavaScript/codeGenNodeTestsVb/MyContractName/DTOs/DecimalsOutputDTO.vb
Imports System
Imports System.Threading.Tasks
Imports System.Collections.Generic
Imports System.Numerics
Imports Nethereum.Hex.HexTypes
Imports Nethereum.ABI.FunctionEncoding.Attributes
Namespace StandardToken.MyContractName.DTOs

    <[FunctionOutput]>
    Public Class DecimalsOutputDTO
    
        <[Parameter]("uint8", "", 1)>
        Public Property ReturnValue1 As Byte
    
    End Class

End Namespace
